client/src/components/WarningText.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Box, Paragraph } from 'grommet'
+import { Icon } from 'components/Icon'
+import { Link } from 'components/Link'
+
+export const WarningText = ({
```

Comment:
Can we use the same interface for passing the margins and pads as grommet? I don't see any benefit of passing in an array and mapping that to the sides.

Reply:
Done, thank you!